command/genesis/genesis.go

```diff
@@ -205,15 +241,45 @@ func (c *GenesisCommand) Run(args []string) int {
 			Seal:          []byte{},
 			CommittedSeal: [][]byte{},
 		}
+
 		extraData = make([]byte, ibft.IstanbulExtraVanity)
 		extraData = ibftExtra.MarshalRLPTo(extraData)
 	}
 
+	// Check if any addresses should be prestaked in the dev consensus
+	if consensus == devConsensus {
+		if len(ibftValidators) != 0 {
+			for _, val := range ibftValidators {
+				validators = append(validators, types.StringToAddress(val))
+			}
+		}
+	}
+
 	if bootnodes.AreSet && len(bootnodes.Addrs) < 2 {
 		c.UI.Error("Minimum two bootnodes are required")
 		return 1
 	}
 
+	// constructEngineConfig is a helper method for
+	// parametrizing the consensus configuration, which
+	// can be retrieved at runtime from the consensus module
+	constructEngineConfig := func() map[string]interface{} {
+		if consensus != ibftConsensus {
+			// Dev consensus, return an empty map
+			return map[string]interface{}{}
+		}
+
+		if isPos {
+			return map[string]interface{}{
+				"type": ibft.PoS,
+			}
+		}
+
+		return map[string]interface{}{
+			"type": ibft.PoA,
+		}
```

Comment:
Can we define each map as constant value?

Reply:
Fixed 👍 

c0fa6eb